src/pages/events/details.astro

```diff
@@ -0,0 +1,425 @@
+---
+import Layout from "../../layouts/Layout.astro";
+import Button from "../../components/elements/Button.astro"
+import Icon from "../../components/elements/Icon.astro"
+
+---
+
+<Layout title="Event Details" description="event details page" keywords="technology" image="">
+    <!-- event hero section -->
+    <section class="pt-40 sm:mx-[149px] mx-4">
+        <div class="border rounded-lg sm:h-44 h-[355px] flex sm:flex-row flex-col bg-gray-200">
+            <!-- content div -->
+            <div class="my-[13.5px] sm:w-[783px] w-[343px] sm:h-[152px] h-[176px] sm:order-1 order-2">
+                <!-- breadcrumb on desktop -->
+                <nav class="text-gray-700 text-sm mb-4 sm:inline hidden" aria-label="Breadcrumb">
```

Comment:
We might need this breadcrumb else where in the future, can we change it to a component?